composer.json

```diff
@@ -37,7 +37,8 @@
         "phpstan:core": "phpstan analyse -c phpstan/core.neon",
         "phpstan:module": "phpstan analyse -c phpstan/module.neon --memory-limit 2G",
         "tests-unit": "vendor/bin/phpunit tests",
-        "tests-coverage": "export XDEBUG_MODE=coverage && php -d memory_limit=200M ./vendor/bin/phpunit tests --coverage-html ./tests-coverage",
+        "tests-coverage": "export XDEBUG_MODE=coverage && php -d memory_limit=200M ./vendor/bin/phpunit tests -c phpunit.xml.dist --log-junit ./tests-coverage/log/junit.xml --testsuite mandatory --coverage-html tests-coverage/html --coverage-cobertura tests-coverage/coverage/cobertura.xml --coverage-text=./tests-coverage/coverage.txt",
```

Comment:
Pro HTML verzi bych nechal původní umístění, aby se to nám to nečekaně neduplikovalo.